app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        sum(split_integer(100, 7)) == 100
+    )
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (
+        max(split_integer(555, 555)) == 1
+    )
```

Comment:
same here. How can we be sure that there are equal amount of parts? Also you should test here number, that is devisible by the next number, like 5 and 25. Not the same numbers